src/routes/applicant.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from 'express';
+import { create_post, update_status_patch } from '../controllers/applicant';
+
+const router = express.Router();
+
+router.post('/create', create_post);
+
+router.patch('/:id/status', update_status_patch);
```

Comment:
The API design should be RESTful.

by sending a POST to /applicant we should implicitly know we are creating a new applicant, and by sending PATCH to /applicant/:id we should know that we are modifying an existing applicant. 

So would change post to /
and patch to /

I would also suggest researching a bit about RESTful APIs too as they are pretty much the industry standard to API design.